RxJavaTemplate/buildSrc/src/main/java/Configurations.kt

```diff
@@ -0,0 +1,18 @@
+class Configurations {
+    object Flavor {
```

Comment:
Would it be better if `Flavour`, `BuildType` & `Module` were not a sub class of `Configurations?

Then the gradle files will be easier to read:

Ex: `Congfigurations.Flavor.Production` will be `Flavors.Production` 🙏

Reply:
@Tuubz Make sense 👍 Three deep level is quite hard for readabilty